Small clean up for Propagators. (#577)
open-telemetry/opentelemetry-specification: specification/trace/api.md

`IsRemote` is a boolean flag which returns true if the SpanContext was propagated
from a remote parent.
When extracting a `SpanContext` through the Propagators API, its `IsRemote` flag MUST

> be set to true, whereas its resulting children MUST have it set to false.

Reviewer: ```suggestion
be set to true, whereas the SpanContext of any child spans MUST have it set to false.
```
